changelog.d/5985.bugfix

```diff
@@ -0,0 +1 @@
+Fix synapse workers turinging into a zombie processes if opentracing is enabled in the config but not installed.
```

Comment:
"turinging"

"**a** zombie process**es**"

zombie processes are a specific thing in unix-like OSes (https://en.wikipedia.org/wiki/Zombie_process), and I don't think they are quite what you have. you could just say 'Stop synapse worker processes from hanging' or something like that.

Reply:
I haven't actually been able to replicate the bug as it was described to me, however the fix is a feature so I've changed it to that.